sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -134,8 +139,78 @@ public void terminateWorkflow(String workflowInstanceId, @Nullable Object output
   }
 
   /**
-   * Closes the inner DurableTask client and shutdown the GRPC channel.
+   * Fetches workflow instance metadata from the configured durable store.
+   *
+   * @param instanceId the unique ID of the workflow instance to fetch
+   * @param getInputsAndOutputs <code>true</code> to fetch the workflow instance's
+     inputs, outputs, and custom status, or <code>false</code> to omit them
+   * @return a metadata record that describes the workflow instance and its
+     execution status, or a default instance if no such instance is found.
+   */
+  @Nullable
+  public WorkflowMetadata getInstanceMetadata(String instanceId, boolean getInputsAndOutputs) {
```

Comment:
Dotnet SKD abstracted this as `GetWorkflowStateAsync`.

https://github.com/dapr/dotnet-sdk/blob/master/src/Dapr.Workflow/DaprWorkflowClient.cs#L75

Could it be better to call it `getInstanceState`?

Reply:
Done